collector/collector/config.py

```diff
@@ -25,9 +25,41 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = True
     LOG_FILE = 'handler.log'
 
-    # The maximum retention time for records stored in the database, measured
-    # from the time received by the `collector` app.
-    MAX_WEEK_KEEP_RECORDS = 5
+    # When PURGE_OLD_RECORDS == True then a purging system of old records will
+    # be triggered daily. If this variable is not present, then no purging will be done.
+    # If the purging system is enabled, then the following two variables must be set
+    PURGE_OLD_RECORDS = True
```

Comment:
This value should match the collector default value (see my comment for collector/collector/purge.py).

Reply:
Exactly